src/Modules/DashboardModule/components/Dashboard/Dashboard.Moule.css

```diff
@@ -0,0 +1,27 @@
+.titles {
+  position: relative;
+}
+.titles::before {
+  content: "";
+  position: absolute;
+  top: 0px;
+  left: -16px;
+  width: 5px;
+  height: 100%;
+  display: block;
+  background-color: #ef9b28;
+}
+.iconss {
+  color: var(--iconColor) !important;
```

Comment:
Using !important in many places is considered a bad practice. As it will make it very difficult to update these styles in a later time.